frontend/src/scenes/session-recordings/mobile-replay/transformer/types.ts

```diff
@@ -14,4 +14,8 @@ export interface ConversionContext {
 // StyleOverride is defined here and not in the schema
 // because these are overrides that the transformer is allowed to make
 // not that clients are allowed to request
-export type StyleOverride = MobileStyles & { bottom?: true; backgroundRepeat?: 'no-repeat' | 'unset' }
+export type StyleOverride = MobileStyles & {
+    bottom?: true
+    backgroundRepeat?: 'no-repeat' | 'unset'
+    'z-index'?: number
```

Comment:
same... we weren't getting proper type hinting because of the indirection and were missing this override